### PR TITLE
Update container-step.md regarding delegate selectors

### PR DESCRIPTION
**Pull request by sourabh-awashti**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md [hunk 1]
```diff
@@ -361,6 +361,9 @@ For example, you can set **Value** as an expression and reference the value of s
 </Tabs>
 ```
 
+### Delegate Selector
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md:364]:
```suggestion
### Delegate selector
```

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md [hunk 2]
```diff
@@ -361,6 +361,9 @@ For example, you can set **Value** as an expression and reference the value of s
...
+Infrastructure delegate selector will be considered for step execution by default unless overridden by step, stage or pipeline level delegate selector.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md:365]:
```suggestion
By default, the infrastructure delegate selector is considered for step execution unless overridden by a step, stage, or pipeline level delegate selector.
```

**Review by roshnisarangadharan — APPROVED** [review]

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md [hunk 3]
```diff
@@ -361,6 +361,9 @@ For example, you can set **Value** as an expression and reference the value of s
 </Tabs>
 ```
 
+### Delegate selector
+By default, the infrastructure delegate selector is considered for step execution unless overridden by a step, stage, or pipeline level delegate selector.
```
Comment (yogesh-chauhan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md:365]: can we say infrastructure connector delegate selector ?

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md [hunk 4]
```diff
@@ -361,6 +361,9 @@ For example, you can set **Value** as an expression and reference the value of s
 </Tabs>
 ```
 
+### Delegate selector
+By default, the infrastructure connector delegate selector is considered for step execution unless overridden by a step, stage or pipeline level delegate selector.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md:365]:
```suggestion
By default, the infrastructure connector [delegate selector](/docs/platform/delegates/manage-delegates/select-delegates-with-selectors/) is considered for step execution unless overridden by a step, stage or pipeline level delegate selector.
```

**Review by roshnisarangadharan — APPROVED** [review]

File: docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md [hunk 5]
```diff
@@ -361,6 +361,9 @@ For example, you can set **Value** as an expression and reference the value of s
 </Tabs>
 ```
 
+### Delegate selector
+By default, the infrastructure connector [delegate selector](/docs/platform/delegates/manage-delegates/select-delegates-with-selectors/) is considered for step execution unless overridden by a step, stage or pipeline level delegate selector.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/executions/cd-general-steps/container-step.md:365]:
```suggestion
By default, the [infrastructure connector delegate selector](/docs/platform/delegates/manage-delegates/select-delegates-with-selectors/) is considered for step execution unless overridden by a step, stage or pipeline level delegate selector.
```

**Review by roshnisarangadharan — APPROVED** [review]

**Review by yogesh-chauhan — APPROVED** [review]

**Pull request merged**